src/App.js

```diff
@@ -4,12 +4,22 @@ import { useState } from 'react';
 
 function App() {
   let fiveContacts = contactsArr.slice(0, 5)
-  console.log(fiveContacts);
+  // console.log(fiveContacts);
   const [contacts, setContacts] = useState(fiveContacts);
+  const newContactsArr = contactsArr.slice(5);
+  // console.log(newContactsArr);
+  const randomIndex = Math.floor(Math.random() * newContactsArr.length);
+  const newContact = newContactsArr[randomIndex];
 
+  const randomContact = () => {
+    setContacts([...fiveContacts, newContact])
+  }
```

Comment:
To make code more readable, it is always good practice to separate code in clear block/functions, so here I would suggest having lines 9,11,12 inside randomContact function

Reply:
Okay. I see